src/core/rbuscore.c

```diff
@@ -1900,7 +1900,9 @@ rbusCoreError_t rbus_discoverObjectElements(const char * object, int * count, ch
     {
         int32_t size, length, i;
         const char * value = NULL;
+        char **array_ptr = NULL;
 
+        *elements = array_ptr;
```

Comment:
This is right but can we explicitly set ``` *elements = NULL;```

Reply:
since *elements = array_ptr; was set earlier *element is pointing to array_ptr ; setting array_ptr to NULL should make *element point to NULL, 
If wrong please correct me